maat-court-data-api/src/main/java/gov/uk/courtdata/model/CreateRepOrder.java

```diff
@@ -0,0 +1,86 @@
+package gov.uk.courtdata.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Data
+@SuperBuilder
```

Comment:
As this class doesn't extend anything I don't think the SuperBuilder does anything?

Reply:
I've replaced this with a normal @Builder now.